demos/background_subtraction_demo/python/background_subtraction_demo.py

```diff
@@ -176,14 +213,21 @@ def main():
     elif args.adapter == 'ovms':
         model_adapter = OVMSAdapter(args.model)
 
-    labels = ['__background__', 'person'] if args.labels is None else args.labels
+    if args.labels is None:
+        labels = ['__background__', 'person']
+    else:
+        with open(args.labels, 'rt') as labels_file:
+            labels = labels_file.read().splitlines()
+        assert len(labels), 'The file with class labels is empty'
```

Comment:
As I understand, the labels are used only for instance-segmentation models. Moreover, the demo does not actually need the whole classes. Only `person` class should be presented in labels variable to define the person_id in order to get the correct mask from instance-segmentation model for visualization.  

If so, we can replace the `--labels` option by `--person_id` option with default value 1, which will be passed directly to `render_results()`. What do you think?

Or you might left the `--labels` option, but in this case I believe the `args.labels` should be included in configuration, and the Instance Segmentation wrappers should load the labels in constructor in the same way, as it done in segmentation wrapper.

Reply:
I think that at least the logic may be like in other demos (for example [segmentation_demo](https://github.com/openvinotoolkit/open_model_zoo/blob/master/demos/segmentation_demo/python/segmentation_demo.py)) and it is easier to find or make text file with labels than calculate id of an interesting one.

Reply:
Okay, let's take this option. Please, align the `args.labels` with segmentation_demo.py

Reply:
Used `load_labels` function.